src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/GlobalRunSpec.cs

```diff
@@ -103,7 +103,7 @@ public void SetupFileSource(FileSourceMatchInfo source)
             Include = new List<IPathMatcher>() { new FileSourceStateMatcher(FileDispositionStates.Include, matcher) };
             Exclude = new List<IPathMatcher>() { new FileSourceStateMatcher(FileDispositionStates.Exclude, matcher) };
             CopyOnly = new List<IPathMatcher>() { new FileSourceStateMatcher(FileDispositionStates.CopyOnly, matcher) };
-            Rename = source.Renames ?? new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+            Rename = source.Renames ?? new Dictionary<string, string>(StringComparer.Ordinal);
```

Comment:
This one makes me a little nervous since it's a change in behavior, but it's probably the right move